src/components/inversion_component/inversion.sh

```diff
@@ -80,9 +80,13 @@ run_inversion() {
         cd ${RunDirs}/inversion
     fi
 
+    # Set inversion memory, CPUs, and time
+    InvMem="${InversionMemory:-$RequestedMemory}"
+    InvCPU="${InversionCPUs:-$RequestedCPUs}"
+    InvTime="${InversionTime:-$RequestedTime}"
     # Execute inversion driver script
-    sbatch --mem $RequestedMemory \
-           -c $RequestedCPUs \
+    sbatch --mem $InvMem \
+           -c $InvCPU \
            -t $RequestedTime \
```

Comment:
Should this be `InvTime`?

Reply:
I think `InversionTime` also needs to be set in the config files. Unless we just use one `RequestedTime` for everything

Reply:
Thanks @megan-he, updated!